test/neura/ctrl/branch_for.mlir

```diff
@@ -100,197 +100,182 @@ func.func @loop_test() -> f32 {
 // CANONICALIZE-NEXT:     "neura.return"(%13) : (!neura.data<f32, i1>) -> ()
 // CANONICALIZE-NEXT:   }
 
-// CTRL2DATA:       func.func @loop_test() -> f32 attributes {accelerator = "neura"} {
+// CTRL2DATA: func.func @loop_test() -> f32 attributes {accelerator = "neura"} {
```

Comment:
The original spaces between `// CTRL2DATA:` and  `func.func` are meant to align the next line. Plz keep them **for all test files**.